docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValues.cs

```diff
@@ -45,49 +36,41 @@ public static void Example()
             // We can extract the newly created column as an IEnumerable of SampleDataTransformed, the class we define below.
             var defaultRowEnumerable = mlContext.Data.CreateEnumerable<SampleDataTransformed>(defaultTransformedData, reuseRowObject: false);
 
-            // a small printing utility
-            Func<object[], string> vectorPrinter = (object[] vector) =>
-            {
-                string preview = "[";
-                foreach (var slot in vector)
-                    preview += $"{slot} ";
-                return preview += "]";
-
-            };
-
             // And finally, we can write out the rows of the dataset, looking at the columns of interest.
-            foreach (var row in meanRowEnumerable)
-            {
-                Console.WriteLine($"Label: {row.Label} Features: {vectorPrinter(row.Features.Cast<object>().ToArray())} MissingReplaced: {vectorPrinter(row.MissingReplaced.Cast<object>().ToArray())}");
-            }
+            foreach (var row in defaultRowEnumerable)
+                Console.WriteLine($"Features: [{string.Join(", ", row.Features)}]\t MissingReplaced: [{string.Join(", ", row.MissingReplaced)}]");
 
             // Expected output:
-            // Notice how the NaN of the Features column for the second row is replaced by the mean of (1, 2, 6) the values in that row
-            // 
-            //Label: 3  Features: [1 1    0] MissingReplaced: [1  1  0]
-            //Label: 32 Features: [0 NaN  1] MissingReplaced: [0  3  1]
-            //Label: 5  Features: [-1 2 - 3] MissingReplaced: [-1 2 -3]
-            //Label: 9  Features: [-1 6 - 3] MissingReplaced: [-1 6 -3]
+            // Features: [∞, 1, 0]      MissingReplaced: [∞, 1, 0]
+            // Features: [0, NaN, 1]    MissingReplaced: [0, 0, 1]
+            // Features: [-1, 2, -3]    MissingReplaced: [-1, 2, -3]
+            // Features: [-1, NaN, -3]  MissingReplaced: [-1, 0, -3]
+
+            // Mean ReplaceMode:
```

Comment:
>// Mean ReplaceMode: [](start = 12, length = 20)

Isn't there one more mode?

Reply:
There is also maximum and minimum, but I don't think it adds much to add them to this sample.

---
In reply to: [273727261](https://github.com/dotnet/machinelearning/pull/3216#discussion_r273727261) [](ancestors = 273727261)

Reply:
For the documentation, if we have more than one, we should have all of them &mdash; right now, it looks inconsistent or incomplete.

---
In reply to: [273735371](https://github.com/dotnet/machinelearning/pull/3216#discussion_r273735371) [](ancestors = 273735371,273727261)

Reply:
I would rather remove the mean replacement mode in this sample then.
We are not trying to exhaust all the possible settings in our samples. Would that be fine?

---
In reply to: [273736506](https://github.com/dotnet/machinelearning/pull/3216#discussion_r273736506) [](ancestors = 273736506,273735371,273727261)